app/plugins/Files/index.html

```diff
@@ -132,6 +138,39 @@
 				</div>
 			</div>
 		</div>
+		<!-- Sliding New Directory Frame -->
+		<div class='frame hidden' id='add-dir'>
+			<div class='header'>
+				<div class='button' id='back-dir'>
+					<i class='fa fa-chevron-left'></i>
+				</div>
+				<div class='title'>
+					Upload Directory
+				</div>
+			</div>
+
+			<div class='pure-g step' id='step1'>
```

Comment:
HTML element id's should be unique. Now there are two elements with `id='step1'` which would make `document.getElementById('step1')` work improperly.